code/modules/mob/living/carbon/human/death.dm

```diff
@@ -13,11 +13,12 @@
 
 		playsound(src.loc, 'sound/effects/gib.ogg', 100, 1, 10)
 
-	for(var/obj/item/organ/I in internal_organs)
+	for(var/obj/item/organ/internal/I in internal_organs)
 		if(istype(loc,/turf))
-			I.removed()
-			spawn()
-				I.throw_at(get_edge_target_turf(src,pick(alldirs)),rand(1,3),5)
+			I.remove(src)
+			I.forceMove(get_turf(src))
+			//spawn()

```

Comment:
this spawn needs to be here, or else if one organ gets thrown into space....the second one won't throw until the first one is done travelling....likewise, the rest of the proc won't complete, either.